lib/logstash/inputs/snmp/client.rb

```diff
@@ -104,20 +104,24 @@ def walk(oid, strip_root = 0)
 
     private
 
+    NULL_STRING = "null".freeze
+
     def coerce(variable)
       variable_syntax = variable.getSyntax
       # puts("variable.getSyntaxString=#{variable.getSyntaxString}")
       case variable_syntax
-      when BER::OCTETSTRING
+      when BER::OCTETSTRING, BER::BITSTRING
```

Comment:
is there any place we can find the whole list of possible options?

Reply:
https://www.snmp4j.org/doc/org/snmp4j/asn1/BER.html